guides/common/modules/proc_enabling-the-project-client-name-repository-for-rhel-7-and-rhel-6.adoc

```diff
@@ -0,0 +1,59 @@
+[id="enabling-the-project-client-name-repository-rhel-7-and-rhel-6"]
+= Enabling the {project-client-name} repository for {RHEL} 7 and {RHEL} 6
+
+[NOTE]
+====
+You require *{RHEL} Extended Lifecycle Support (ELS) Add-on* subscription to enable the repositories of {RHEL} 6.
+For more information, see https://www.redhat.com/en/resources/els-datasheet[{RHEL} Extended Lifecycle Support (ELS) Add-on] guide.
+====
+
+To use the CLI instead of the {ProjectWebUI}, see the procedure relevant for your {RHEL} version:
+
+* xref:CLI_Enabling_the_Client_Repository_rhel_7_{context}[]
+* xref:CLI_Enabling_the_Client_Repository_rhel_6_{context}[]
+
+ifeval::["{mode}" == "disconnected"]
+.Prerequisites
+* Ensure that you import all content ISO images that you require into {ProjectServer}.
+endif::[]
+.Procedure
+. In the {ProjectWebUI}, navigate to *Content* > *Red Hat Repositories*.
+. In the *Available Repositories* pane, enable the *Recommended Repositories* to get the list of repositories.
+. In the *Available Repositories* pane, click on *{project-client-name} (for RHEL 7 Server) (RPMs)* or *{project-client-name} (for RHEL 6 Server - ELS) (RPMs)* to expand the repository set.
```

Comment:
We have an attribute for "-": https://github.com/theforeman/foreman-documentation/blob/master/guides/common/attributes-typography.adoc 

See also all other product names in various Hammer procedures

Reply:
I don't think it's necessary to update old procedures with "-" attributes (and these procedures are old, as explained in the PR description). So no change.